### ISSUE TITLE
How to visualize Ground Truth Mask of own Dataset with multiple classes?

### ISSUE THREAD

**Issue by FloBr49** [matterport/Mask_RCNN#2567]
Hi everyone, 

i wanted to know how i can visualize the Ground Truth Mask of my training dataset. I have a dataset with 8 classes and in every image are multiple different classes. Is it possible to generate a ground truth mask, where i can see all instances?

Thank you very much.
